bbs/menus/menusupp.cpp

```diff
@@ -291,7 +302,7 @@ void GoodBye() {
     int cycle = 0;
     do {
       bout.cls();
-      bout.printfile(filename.string());
+      bout.printfile_path(filename.string());
```

Comment:
actually please remove .string() from filename, since filename here is a std::filesystem::path (which is what printfile_path wants)